Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode {
+    stage('Install Dependencies') {
+        npmInstall {}
+    }
+
+    stage('Lint') {
+        print 'Lint task is broken see https://issues.jboss.org/browse/FH-3611'
+        //sh 'grunt eslint'
+    }
```

Comment:
Does this really fail? Did you try it? If it does, you should create a different jira for it.